### ISSUE TITLE
Cannot Connect Error

### ISSUE THREAD

**Issue by daveyhellstormelite** [tweakunwanted/OpenXC-Resellers#7]
Getting error cannot connect tried a remote server and same server my XC panel is on

**Comment by tweakunwanted** [comment]
Did you run install.php?

**Comment by daveyhellstormelite** [comment]
yes tells me to make sure config file is correct in the config file do I change the port from 7999 or leave as is thanks

**Comment by FerrahWolfeh** [comment]
It depends on where are you running the office panel
If you're running on a server that's different from the main admin panel, you'll have to give permissions for the panel to access the remote database

**Comment by FerrahWolfeh** [comment]
If it is on a different server, you'll have to go into your root (or given user in config.php) user in the main server's mysql and type

grant all on xtream_iptvpro.* to root@(office server ip) identified by 'password'; .
Then you type

flush privileges; 

and try

**Comment by FerrahWolfeh** [comment]
In the 'password' you put the db password (with quotes) that you wrote on Config.php

**Comment by daveyhellstormelite** [comment]
im running it on same server for testing first but cannot get it to work

**Comment by FerrahWolfeh** [comment]
So, let's start from the beginning...
What appears when you enter install.php

**Comment by FerrahWolfeh** [comment]
And, give me a screenshot of your config.php

**Comment by daveyhellstormelite** [comment]
i get the check config.php on `install

<?php
	/* SERVER SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO SERVIDOR */
	define("OFFICE_KEY", "YOUR_LICENCE_KEY");

	/* RESELLERS PANEL DATABASE SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO BANCO DE DADOS DO PAINEL REVENDEDOR */
       define("OFFICE_DB_HOST", "localhost");
       define("OFFICE_DB_PORT", "3306");
       define("OFFICE_DB_NAME", "openxc_resellers");
       define("OFFICE_DB_USER", "root");
       define("OFFICE_DB_PASS", "PASSWORD");

 	/* XTREAM PANEL DATABASE SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO BANCO DE DADOS DO SERVIDOR XTREAM CODES */
	define("DB_HOST", "127.0.0.1");
	define("DB_PORT", "7999");
	define("DB_NAME", "xtream_iptvpro");
	define("DB_USER", "root");
	define("DB_PASS", "PASSWORD");

	/* REPLACE WITH YOUR DOMAIN OR IP BUT KEEP /i */ /* SUBSTITUA COM O SEU ENDEREÃ‡O IP OU DOMINIO MATENHA O /i */
	define("SHORTENER_URL", "http://MY-IP/i");

	/* ALLOWED EMAILS FOR TESTS */ /* EMAILS PERMITIDOS PARA TESTES */
	define ("ALLOWED_EMAILS", serialize (array ("root@localhost.com", "exemplo2@gmail.com")));

	/* ENABLE DEBUG LOG */
	define("OFFICE_DEBUG", 0);
?>

**Comment by daveyhellstormelite** [comment]
`<?php
	/* SERVER SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO SERVIDOR */
	define("OFFICE_KEY", "YOUR_LICENCE_KEY");

	/* RESELLERS PANEL DATABASE SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO BANCO DE DADOS DO PAINEL REVENDEDOR */
       define("OFFICE_DB_HOST", "localhost");
       define("OFFICE_DB_PORT", "3306");
       define("OFFICE_DB_NAME", "openxc_resellers");
       define("OFFICE_DB_USER", "root");
       define("OFFICE_DB_PASS", "PASSWORD");

 	/* XTREAM PANEL DATABASE SETTINGS */ /* CONFIGURAÃ‡Ã•ES DO BANCO DE DADOS DO SERVIDOR XTREAM CODES */
	define("DB_HOST", "127.0.0.1");
	define("DB_PORT", "7999");
	define("DB_NAME", "xtream_iptvpro");
	define("DB_USER", "root");
	define("DB_PASS", "PASSWORD");

	/* REPLACE WITH YOUR DOMAIN OR IP BUT KEEP /i */ /* SUBSTITUA COM O SEU ENDEREÃ‡O IP OU DOMINIO MATENHA O /i */
	define("SHORTENER_URL", "http://MY-IP/i");

	/* ALLOWED EMAILS FOR TESTS */ /* EMAILS PERMITIDOS PARA TESTES */
	define ("ALLOWED_EMAILS", serialize (array ("root@localhost.com", "exemplo2@gmail.com")));

	/* ENABLE DEBUG LOG */
	define("OFFICE_DEBUG", 0);
?>
`

**Comment by daveyhellstormelite** [comment]
i have created DB and imported the 2 sql files

**Comment by FerrahWolfeh** [comment]
Hmm, well, assuming that you placed the configs correctly, and the office is installed on the same server as the OpenXC, try to change the panel db port to 7999 and put you user, etc. Then tell me what happens?

**Comment by daveyhellstormelite** [comment]
still get the same page in install.php I change port to 7999 and checked in mysql if it change which it did after mysql restart.

**Comment by FerrahWolfeh** [comment]
Ok, just to clear my consciousness, try to install php-mysqli, sometimes it's not installed with php alone

**Comment by daveyhellstormelite** [comment]
fixed it mate it was mysql problems wouldnt let me login with phpmyadmin part either
